### ISSUE TITLE
"A strongly-named assembly is required"

### ISSUE THREAD

**Issue by goswinr** [fsprojects/FsXaml#66]
using" F# Empty Windows App (WPF)"  template ( version 2.0):
I create an new project and update all nuget packages.
I then get the error "A strongly-named assembly is required" on the type provider.
see images.
Is this a problem with my setup ? Is this something I can fix directly?

(using VS2015 update 3)
![2017-03-16_174025](https://cloud.githubusercontent.com/assets/884885/24007959/77cf6cb6-0a70-11e7-9c39-07a7b95b30bb.jpg)
![2017-03-16_173955](https://cloud.githubusercontent.com/assets/884885/24007969/7b7cae50-0a70-11e7-9a66-aebc6537f42f.jpg)

**Comment by ReedCopsey** [comment]
Are you using F# Power Tools?  Try temporarily disabling it, and see if it
goes away...

**Comment by goswinr** [comment]
Yes, the Error goes away after disabling the power tools. Is that the only solution ?

**Comment by ReedCopsey** [comment]
Temporarily, yes.  Or ignore it (it should build -it's an "intellisense"
issue).  I'm working on a pull request for power tools to eliminate this
issue permanently, but it's a large job, and not completed yet.

**Comment by goswinr** [comment]
I have some other errors now but i think I can mange those. seems the FsViewModule API has changed.
![2017-03-16_180713](https://cloud.githubusercontent.com/assets/884885/24009005/cef595e4-0a73-11e7-9491-fcf3e4cf9926.jpg)

**Comment by ReedCopsey** [comment]
Yes, change the opens to:

```
open ViewModule
open ViewModule.FSharp
```

And the issue should go away.  You also don't need the ", true" on line 9
anymore.  Both of these libraries changed APIs in their last major releases.

FYI - We're planning to write our own templates for FsXaml, in this
repository... at which time the community templates (which are horribly out
of date) should be unnecessary.

**Comment by goswinr** [comment]
OK, many thanks for your fast reply Reed!

**Comment by brianberns** [comment]
Just curious what the status of this issue is. Since it interferes with Intellisense, it makes development with FsXaml much more difficult. (And I don't want to disable F# Power Tools, since that would also make development more difficult.)

**Comment by gdennie** [comment]
Developing on VS2015 targeting .net framework v4.6.1 and runtime FSharp.Core v4.4.0, I was able to get FsXaml to work by replacing v3.1.6 with v3.0.0.
